packages/gatsby-source-vtex/src/redirects.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs'
```

Comment:
Maybe you need lint this file

Reply:
my `linter`, and `git staged` are not complaining about this file. What's your linter is complaining about ?